kube/services/jobs/workspace-launch-test-cronjob.yaml

```diff
@@ -0,0 +1,197 @@
+# To run: gen3 job run workspace-launch-test-cronjob COMMONS_URL <commons base url> NOTEBOOK <NOTEBOOK name>
+# example: gen3 job run workspace-launch-test-cronjob COMMONS_URL https://qa-heal.planx-pla.net NOTEBOOK '(Generic) Jupyter Lab Notebook with R Kernel'
+# If NOTEBOOK isn't provided, it defaults to '(Generic) Jupyter Lab Notebook with R Kernel'
```

Comment:
So this means this cronjob can only test one selected workspace launch a time right? I would see if this fits @jawadqur 's need
I think it would be better if we can mount a list of workspace names into this job, and triggers job script over this list of workspace names. This might be easier for using it, but if it is significant harder we can also defer it to another ticket

Reply:
I see what you mean. I think it shouldn't be too difficult to change